### PR TITLE
Maven task

### PR DESCRIPTION
**Pull request by KrasimirRaikov**
four modules deployed with maven, using the jarjar library.

### CODE REVIEW

File: myproject-bl/src/main/resources/log4j2.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (panayotkulchev) [inline myproject-bl/src/main/resources/log4j2.xml]: I this file needed.

File: myproject-bl/target/classes/log4j2.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (panayotkulchev) [inline myproject-bl/target/classes/log4j2.xml]: And this one is needed?

File: myproject-pl/pom.xml [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+  </build>
+
+  <dependencies>
+    <dependency>
+      <groupId>org.postgresql</groupId>
+      <artifactId>postgresql</artifactId>
+      <version>9.4.1208.jre7</version>
+    </dependency>
+
+    <dependency>
```
Comment (panayotkulchev) [inline myproject-pl/pom.xml]: Is this dependency needed in case you already added in the main project pom.xml

Comment (panayotkulchev) [inline myproject-pl/pom.xml]: Is this dependency needed in case you already added in the main project pom.xml

**Comment by panayotkulchev** [comment]
After fix my comments you have LGTM.

**Comment by panayotkulchev** [comment]
No label needs review was added.

File: myproject-pl/pom.xml (hunk 3 above)
Comment (KrasimirRaikov) [inline myproject-pl/pom.xml]: removed it.

File: myproject-bl/src/main/resources/log4j2.xml (hunk 1 above)
Comment (KrasimirRaikov) [inline myproject-bl/src/main/resources/log4j2.xml]: it wasn't, i removed it.

File: myproject-bl/target/classes/log4j2.xml (hunk 2 above)
Comment (KrasimirRaikov) [inline myproject-bl/target/classes/log4j2.xml]: removed it.

**Comment by panayotkulchev** [comment]
LGTM!

**Comment by panayotkulchev** [comment]
Do not forget to squash before merge!

**Pull request merged**